src/fhirtypes/ElementDefinition.ts

```diff
@@ -602,6 +615,192 @@ export class ElementDefinition {
     };
   }
 
+  fixValue(value: FixedValueType): void {
+    if (typeof value === 'boolean') {
+      this.fixBoolean(value);
+    } else if (typeof value === 'number') {
+      this.fixNumber(value);
+    } else if (typeof value === 'string') {
+      this.fixString(value);
+    } else if (value instanceof Code) {
+      this.fixFshCode(value);
+    } else if (value instanceof FshQuantity) {
+      this.fixFshQuantity(value);
+    } else if (value instanceof FshRatio) {
+      this.fixFshRatio(value);
+    }
+  }
+
+  private checkIfFixed(
```

Comment:
This is misleadingly named. I would expect it to return `true` if it *is* fixed. It seems like it does the opposite.

Reply:
Perhaps rename it to `checkIfCanBeFixed` or `checkIfFixable`